css/style.css

```diff
@@ -0,0 +1,589 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+}
+
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.header-main {
+  width: 100%;
+  height: auto;
+  border: 4px solid yellow;
+}
+
+ul,
+ul li {
+  list-style-type: none;
+}
+
+.navbar-toggler-icon {
+  display: none;
+}
```

Comment:
- [ ] You have this class but you don't use it in your HTML. Please delete it. Also, you can do the same behavior of display: none; with bootstrap classes (d-none) if you need it